behave_django/management/commands/behave.py

```diff
@@ -69,6 +72,18 @@ def add_behave_arguments(parser):  # noqa
         '--simple',
     ]
 
+    def validate_runner(value):
+        try:
+            return import_string(value)
+        except ImportError:
+            raise ArgumentTypeError("No module named '%s' was found." % value)
```

Comment:
While the error message is technically correct it doesn't provide context that helps a developer to figure out what they did wrong and what is needed to fix the situation.

Also, `validate_runner` is a very generic name. The code is very specifically about trying to import a module, isn't it?

Reply:
Seems pretty clear to me, i show them what they typed and tell them its wrong. What do you suggest?